tests/conftest.py

```
@@ -77,11 +79,39 @@ def _ensure_async_create_task_mock(real: Any, side_effect: Any) -> None:
+        raising=False,
+    )
+    monkeypatch.setattr(
+        _helpers_mod,
+        "async_create_clientsession",
+        create_clientsession,
+        raising=False,
+    )
+    monkeypatch.setattr(
+        _helpers_mod.aiohttp,
+        "CookieJar",
+        lambda *a, **k: object(),
```

Ruff (0.15.18)

[warning] Unused lambda argument: `a`

(ARG005)